### PR TITLE
add B2C test for hybrid and code flow

### PR DESCRIPTION
**Pull request by lovemaths**

### CODE REVIEW

File: test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+2xFZWpsUU/jtoVCwIkaqZuo6xoKtlMYXXvfVHGuKBHEBVn8b8x/57BQWz1d0KdrN\n\
+XxuMvtFe6RzMqiMqzqZrzae4UqVCkYqcR9gQx66Ehq7hPmCxJCkg7ajo7fu6E7dP\n\
+d34KH2HSYRsaaEA/BcKTeb9H1XE/qEKjog68wUU9Ekfl3FBIRN+1Ah/BoktGFoXy\n\
+i/jt0+L0+gKcL1BLmUlGzMusvRbjI/0+qj+mc0utGdRjY+xIN2yBj8vl4DODO+wM\n\
+wfp+cqZbCd9TENyHaTb8iA27s+73L3ExOQIDAQAB\n\
+-----END RSA PUBLIC KEY-----\n";
+
+
+/*
+ * test strategy (for response_type = 'id_token') which checks the expiration of id_token
```
Comment (polita) [inline test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js]: Update comment?

File: test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+ */
+
+var options = {
+  redirectUrl: 'https://localhost:3000/auth/openid/return',
+  clientID: 'f0b6e4eb-2d8c-40b6-b9c6-e26d1074846d',
+  identityMetadata: 'https://login.microsoftonline.com/mytenant.onmicrosoft.com/v2.0/.well-known/openid-configuration',
+  responseType: 'code id_token',
```
Comment (polita) [inline test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js]: I see you're testing the hybrid flow here. The PR indicates you're testing the code flow too. Where do you test responseType: 'code'?

File: test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js (hunk 1 above)
Comment (lovemaths) [inline test/Chai-passport_test/b2c_oidc_hybrid_and_code_flow_test.js]: done

**Comment by polita** [comment]
:shipit:

**Pull request merged**